developer_manual/design/layoutcomponents.rst

```diff
@@ -95,7 +95,7 @@ Settings
 
 `Settings Vue component <https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppNavigation?id=ncappnavigationsettings>`_.
 
-`Penpot Modals <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=3f784c86-6c27-80c6-8002-6ab157b6bd27&section=interactions&index=12&share-id=88d87192-d205-8007-8002-8d8d21a0dcca>`_
+`Penpot modals <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=3f784c86-6c27-80c6-8002-6ab157b6bd27&section=interactions&index=12&share-id=11fde340-21f4-802e-8002-8d8d305e7ab5>`_
```

Comment:
@nimishavijay on the modals page, the link to the docs doesn’t seem clickable.
Do you think we should also label the modals according to size in Penpot? But probably it’s enough in the docs.

Reply:
Ah you're right. The links didn't save for some reason? I will check again and fix it. 

We can add labels for sizes on Penpot too. It would be similar to the labels for the Checkbox/radio/switch component :)